examples/3d/perspective_zoom.rs

```diff
@@ -0,0 +1,125 @@
+//! Shows how to zoom and orbit a perspective projection camera.
```

Comment:
Could you cross-reference this and the first person view model example? Both feature perspective zoom.